### ISSUE TITLE
Relative path for `template` option does not work

### ISSUE THREAD

**Issue by varya** [styleguidist/react-styleguidist#211]
In my project, I store `styleguide.config.js` not in the root but in `client-src/webpack` folder.

For the components and the assets directories I needed to use relative path. But for the `template` option, the tool wants me to have a path from the root of the project. It looks very confusing.

This is the content of my config:

```js
const path = require("path");
const glob = require('glob');

const siteWebpack = require("./webpack.config.js");

module.exports = {
  title: 'Awesome styleguide',
  components: '../react/components/**/*.jsx',
  updateWebpackConfig(webpackConfig) {
    siteWebpack.module.loaders.forEach(function(loader) {
      webpackConfig.module.loaders.push(loader);
    });
    webpackConfig.module.noParse = siteWebpack.module.noParse;
    siteWebpack.plugins.forEach(function(plugin) {
       webpackConfig.plugins.push(plugin);
    });
    webpackConfig.postcss = siteWebpack.postcss;

    return webpackConfig;
  },
  assetsDir: '../../assets',
  template: './client-src/webpack/styleguide.template.html'
};
```

**Comment by sapegin** [comment]
I thinks it’s easy to change ;-) You can try if you have time, or I’ll fix it later.

**Comment by sapegin** [comment]
It should be fixed in the 5.0.0 beta. Could you please try it?

https://github.com/styleguidist/react-styleguidist/releases/tag/5.0.0%40beta.9

**Comment by sapegin** [comment]
Closing this for now. Feel free to reopen if you still have this issue.
